puppet/modules/scaleway/manifests/cron.pp

```diff
@@ -0,0 +1,15 @@
+# Deploys a cronjob which calls the Scaleway API to
+# snapshot a server. Does not need to be on the Scaleway host.
+#
+define scaleway::cron (
+  String $api_key = 'NA',
+  String $org_id  = 'NA'
+) {
+  file { "/etc/cron.daily/scaleway-snapshot-${name}":
```

Comment:
The downside is that there is no purging of unmanaged crons now. You probably want to make ensure a parameter.

Reply:
sounds good, will do